mailbox/cassandra/src/test/java/org/apache/james/mailbox/cassandra/mail/CassandraMessageDAOTest.java

```diff
@@ -138,8 +141,23 @@ public void saveShouldStoreMessageWithAttachmentButNoCid() throws Exception {
         assertThat(attachmentRepresentation.get(0).get().getCid().isPresent()).isFalse();
     }
 
-    private SimpleMailboxMessage createMessage(MessageId messageId, String content, int bodyStart, PropertyBuilder propertyBuilder, List<MessageAttachment> attachments) {
-        return new SimpleMailboxMessage(messageId, new Date(), content.length(), bodyStart, new SharedByteArrayInputStream(content.getBytes()), new Flags(), propertyBuilder, MAILBOX_ID, attachments);
+    private SimpleMailboxMessage createMessage(
+        MessageId messageId,
+        String content,
+        int bodyStart,
+        PropertyBuilder propertyBuilder,
+        List<MessageAttachment> attachments
+    ) {
+        return new SimpleMailboxMessage(
```

Comment:
I did not know that before I arrive in this team. It is because of you. But when I think about it builder are a lot of boiler code. It would be great to use something like that: https://immutables.github.io/